src/CMakeLists.txt

```diff
@@ -34,7 +34,10 @@ if(MSVC)
   target_compile_options(backward PUBLIC /wd4267)
   target_compile_options(backward PUBLIC /wd4996)
 endif()
-install (TARGETS backward DESTINATION ${LIB_INSTALL_DIR})
+install (TARGETS backward 
+         LIBRARY DESTINATION ${LIB_INSTALL_DIR}
+         ARCHIVE DESTINATION ${LIB_INSTALL_DIR} 
+         RUNTIME DESTINATION ${BIN_INSTALL_DIR})
```

Comment:
@j-rivero , do you foresee any problems on Linux or macOS?

Reply:
According to [documentation](https://cmake.org/cmake/help/latest/command/install.html):
```
RUNTIME

    Target artifacts of this kind include:

        Executables (except on macOS when marked as MACOSX_BUNDLE, see BUNDLE below);
        DLLs (on all Windows-based systems including Cygwin; note that the accompanying import libraries are of kind ARCHIVE).
```
Should affect only DLLs on Windows being backward a library. I don't see problems here.